<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Find</class>
 <widget class="QDialog" name="Find">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="1">
      <widget class="QCheckBox" name="_cbMatchWord">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="_cbRegex">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QPushButton" name="_btnClose">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="_cbMatchCase">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="_lbMatchCase">
       <property name="text">
        <string>Match Case</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="_lbMatchWord">
       <property name="text">
        <string>Match whole word</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="_lbRegex">
       <property name="text">
        <string>Use regex</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="_lbWrapAround">
       <property name="text">
        <string>Wrap around</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="_cbWrapAround">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="5" rowspan="2">
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="_leSearch"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="_lbSearch">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Search for :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="_lbWarning">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;End of Document&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="_btnNext">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="_btnPrev">
       <property name="text">
        <string>Prev</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>_leSearch</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
